onnxruntime/test/contrib_ops/multihead_attention_op_test.cc

```diff
@@ -467,7 +476,7 @@ TEST(MultiHeadAttentionTest, CrossAttention_Batch2_HeadSize40) {
 }
 
 TEST(MultiHeadAttentionTest, CrossAttention_Batch2_HeadSize32_RightSidePadding_Mask1D) {
-  ROCM_GTEST_SKIP("ROCm MHA does not support mask");
```

Comment:
MIGX_GTEST_SKIP Here.

Reply:
OK, I just disable all tests for MIGraphX in this case. So we don't worry with it in the future.